### PR TITLE
Create cchow33.json

### PR DESCRIPTION
**Pull request by cchow33**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1755"></a>

### CODE REVIEW

File: public/data/cchow33.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Carmen Chow",
+  "type": "personal",
+  "bio": "Aspiring FE Dev!",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/cchow33",
+      "icon": "github"
+    },
```
Comment (Panquesito7) [inline public/data/cchow33.json:10]: There's an extra comma in here.
```suggestion
    }
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/cchow33.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  ],
+  "milestones": [
+    {
+      "title": "Decided to join a bootcamp",
+      "date": "October 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Looking forward to accelerating my learning!",
```
Comment (Njong392) [inline public/data/cchow33.json:18]: Oops. There seems to be a comma at the end of line 18 that is not required. Feel free to remove it :smiley: 
```suggestion
      "description": "Looking forward to accelerating my learning!"
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by RealSollyM — CHANGES_REQUESTED** [review]
Trailing commas found in JSON file. Please fix and re-submit.

**Comment by EmmaDawsonDev** [comment]
@cchow33 im closing this for now as it seems to have been abandoned. Feel free to make the requested changes and open a new pr if you still wish to contribute

**Pull request closed without merging**